### DISCUSSION TITLE
technology/learn/upyun

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#275] [Announcements]
# technology/learn/upyun

分享一下又拍云的简单用法，以及使用它搭建图床。

https://sugarat.top/technology/learn/upyun.html

**Comment by Sunrisies** [comment]
方便问一下吗？使用七牛云的话，来进行上传视频，好像不能使用

**Reply by ATQQ** [reply]
理论上没有限制，只是这里上传限制了只能是图片格式，要传视频的话需要调整一下代码。
